index.bs

```diff
@@ -1233,6 +1233,38 @@ The {{MLOperand}} objects are created by the methods of {{MLGraphBuilder}}, inte
 To <dfn for="MLGraphBuilder">validate operand</dfn> given {{MLGraphBuilder}} |builder| and {{MLOperand}} |operand|, return true if |operand|.{{MLOperand/[[builder]]}} is |builder|, and false otherwise.
 </p>
 
+## {{MLConstantOperand}} interface ## {#api-mlconstantoperand}
+
+An {{MLConstantOperand}} is an {{MLOperand}} which represents constant weight data. Its {{MLConstantOperand/[[data]]}} is statically known during graph construction time, which allows for optimizations such as weight preprocessing.
+
+<script type=idl>
+[SecureContext, Exposed=(Window, DedicatedWorker)]
+interface MLConstantOperand : MLOperand {};
+</script>
+
+<div class=internal-slots>
+{{MLConstantOperand}} has the following internal slots:
+  <dl dfn-type=attribute dfn-for="MLConstantOperand">
+    : <dfn>\[[data]]</dfn> of type [=byte sequence=]
```

Comment:
Just thinking about how future spec text would specify the actual operation on tensor data... should this just be a list of numbers instead of a byte sequence? Otherwise every access to a tensor would have to invoke an algorithm to get the numbers, and that would need to reference "decoding" the byte sequence. Sure, that can be wrapped in helpers, but is there any benefit to holding the bytes here vs. just a list of numbers?

Reply:
Good thinking. I agree that sounds nice :)

This is making me realize that I've never before had to specify anything touching the contents of an `ArrayBufferView`... For the `constant(descriptor, bufferView)` method, how do I get a `[/list]` of numbers out of the `bufferView`?

Reply:
I'm not finding an example and I can't think of a place outside ECMAScript itself that would need to. You may have to write it yourself, using [GetValueFromBuffer](https://tc39.es/ecma262/#sec-getvaluefrombuffer) which [get a copy of the bytes](https://webidl.spec.whatwg.org/#dfn-get-buffer-source-copy) does.